build.gradle

```diff
@@ -0,0 +1,179 @@
+buildscript {
+    ext {
+        corda_release_group = 'net.corda'
+        corda_release_version = '5.0-SNAPSHOT-WILLH'
```

Comment:
Reminder to change this back when you have your core PRs merged. Is the branch that creates this build available on github? Perhaps we can add instructions in the README for how to do it. E.g. clone corda repo, checkout your branch, install from your branch. Clone this repo, etc.

Reply:
Sounds good - I'll add that to the README.